dockers/docker-ptf/Dockerfile.j2

```diff
@@ -139,6 +139,17 @@ RUN ln -s /usr/bin/tcpdump /usr/sbin/tcpdump
 
 RUN mkdir -p /var/log/supervisor
 
+# add dependencies to build gnmi_get
+RUN wget -O go.tgz https://dl.google.com/go/go1.12.linux-amd64.tar.gz \
+    && tar -C /usr/local -xzf go.tgz \
+    && export PATH=$PATH:/usr/local/go/bin \
+    && go version \
```

Comment:
Is this here so that the version gets output to the build log for debugging purposes?

Reply:
yes just to be sure which version gets installed. Like this go1.12 linux/amd64

Reply:
please use latest stable go lang version, and check the example from sonic-slave dockerfile.

```
RUN export VERSION=1.14.2 \
{%- if CONFIGURED_ARCH == "armhf" %}
 && wget https://storage.googleapis.com/golang/go$VERSION.linux-armv6l.tar.gz \
 && tar -C /usr/local -xzf go$VERSION.linux-armv6l.tar.gz \
{%- elif CONFIGURED_ARCH == "arm64" %}
 && wget https://storage.googleapis.com/golang/go$VERSION.linux-arm64.tar.gz \
 && tar -C /usr/local -xzf go$VERSION.linux-arm64.tar.gz \
{%- else %}
 && wget https://storage.googleapis.com/golang/go$VERSION.linux-amd64.tar.gz \
 && tar -C /usr/local -xzf go$VERSION.linux-amd64.tar.gz \
{%- endif %}
 && echo 'export GOROOT=/usr/local/go' >> /etc/bash.bashrc \
 && echo 'export PATH=$PATH:$GOROOT/bin' >> /etc/bash.bashrc \
 && rm go$VERSION.linux-*.tar.gz
```

Reply:
Thanks for this reference!

Reply:
Updated based on suggestion. And docker builds successfully. Build logs:

2020-06-08 17:30:04 (53.2 MB/s) - 'go1.14.2.linux-amd64.tar.gz' saved [123658438/123658438]

^[[0mtotal 14648
-rwxr-xr-x 1 root root 14998378 Jun  8 17:30 gnmi_get